xml/ha_configuring_resources.xml

```diff
@@ -1235,7 +1235,7 @@ IPMI STONITH external device (stonith:external/ipmi)
 
 ipmitool based power management. Apparently, the power off
 method of ipmitool is intercepted by ACPI which then makes
-a regular shutdown. If case of a split brain on a two-node
+a regular shutdown. If case of a split-brain on a two-node
 it may happen that no node survives. For two-node clusters
 use only the reset method.
```

Comment:
```suggestion
a regular shutdown. In case of a split brain on a two-node,
it may happen that no node survives. For two-node clusters,
use only the reset method.
```